### DISCUSSION TITLE
Display of Radio stream info is off....

### DISCUSSION THREAD

**Discussion by cmh714** [rern/rAudio#1497] [Bug Reports]
Basically, I get the song name twice, instead of song and album. Also mpc shows it correctly so suspect a parsing glitch.....

**Comment by rern** [comment]
Please post the song name showed by mpc.

**Reply by cmh714** [reply]
mpc
RipRockRadio: Robert Cray - Smoking Gun - 20th Century Masters: The Millennium Collection: Best Of Robert Cray
[playing] #2/8   0:19/0:00 (0%)
volume: n/a   repeat: on    random: off   single: off   consume: off

**Reply by rern** [reply]
Each WebRadio stream normally contains only a single metadata, the title.
If it contains ` - ` or `: `, rAudio parse it as `Artist` ` - `/`: ` `Title`. Your radio station puts multiple of both in it which is not normal.

**Reply by cmh714** [reply]
As always, appreciate ya Rern!!!  This is the station: http://streaming.live365.com/a40905

Not sure when or if they changed their format, but this has been working correctly for a long time now and broke a few months ago. If you can fix it great, and if not, its not the biggest deal breaker for me anyways.

Thanks Again!!
